modules/gapi/src/backends/oak/goakbackend.cpp

```diff
@@ -0,0 +1,616 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include <opencv2/gapi/gkernel.hpp> // GKernelPackage
+
+#ifdef WITH_OAK_BACKEND
```

Comment:
Is it the right `#ifdef` by the way? Usually our code checks for `HAVE_` but not `WITH_`. `WITH_` is usually a CMake option.

Reply:
Fixed